addons/barcodes_gs1_nomenclature/models/barcode_nomenclature.py

```diff
@@ -88,15 +88,26 @@ def parse_gs1_rule_pattern(self, match, rule):
             result['value'] = match.group(2)
         return result
 
+    def _convert_gs1_separators(self, barcode):
+        """Convert the barcode to use the standard FNC1 separator"""
+        if self.gs1_separator_fnc1:
+            barcode = re.sub(self.gs1_separator_fnc1, FNC1_CHAR, barcode)
+        if barcode[0] == FNC1_CHAR:
+            # Code128 will have a start character but is otherwise compatible
+            # However parser doesn't support a leading separator so remove
+            # here.
+            barcode = barcode[1:]
+        return barcode
+
     def gs1_decompose_extanded(self, barcode):
-        """Try to decompose the gs1 extanded barcode into several unit of information using gs1 rules.
+        """Try to decompose the gs1 extended barcode into several unit of information using gs1 rules.
 
         Return a ordered list of dict
         """
+        # FIXME: Master, there is no 'a' in extended
```

Comment:
no need to add a comment 
can be fixed on the forward port of this PR when it reaches master